site/components/PodcastCard/index.tsx

```diff
@@ -19,14 +23,19 @@ export function PodcastCard(props: CardProps) {
         <Text t="body2" className="summary">
           {props.podcast.summary}
         </Text>
+        {props.podcast.rssId && (
```

Comment:
ah good point, after we add all the ids, we should assume that the rss id always exists and then enforce it on the content people so they cant publish without it. I guess thats another cms update follow-up!
```suggestion
        {props.podcast.rssId && (
```

Reply:
Yep, have updated all rssId's inside sanity and have now removed this check. In the next PR which will be removing the embed fields and code, I will enforce this to be a required field.